pkg/cmd/info/cmd.go

```diff
@@ -72,23 +72,28 @@ func InfoCommand() *cobra.Command {
 	return cmd
 }
 
-func runCommand(opts *infoOptions) func(*cobra.Command, []string) {
-	return func(cmd *cobra.Command, args []string) {
+func runCommand(opts *infoOptions) func(*cobra.Command, []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		// Don't want to output usage and placeholder error if this command fails. We can't set this higher up
+		// since it will suppress usage and error messages for actual incorrect usage (e.g. wrong number of args)
+		cmd.SilenceUsage = true
```

Comment:
Nope, that still suppresses usage in all cases (no message/error if you use a non-existent command or the wrong number of args).

Reply:
strange. I would assume looping through all the added commands in  [here](https://github.com/jozu-ai/kitops/blob/7b38cf36a100f4786098b2001aa25399d54f979f/cmd/root.go#L92) would not be different than adding them on each file.

Reply:
As far as I understand, it's an execution time switch -- if parsing the command hits an error before it hits a RunE, the error and usage will be printed. Once it hits the RunE and calls the `runCommand(opts)` function, we disable printing usage/errors and so it won't print anything on errors after that point. It would maybe work in the root command's persistentPreRunE but I'm kind of tired of grappling with such a weird decision.

Reply:
Turns out that does work: https://github.com/jozu-ai/kitops/pull/268